### ISSUE TITLE
Please clarify how to install in documentation 

### ISSUE THREAD

**Issue by hectoroso** [p2/OAuth2#12]
Hi, I'm having trouble adding your library to my project. The documentation states "the built framework can be used on ... iOS 8 and later."  What specifically do I add tp my project?

I've downloaded the code and opened it Xcode and I see a folder named Products with two references to Oauth2.framework, both in red, so no files found. Do I need to build something?

I've also looked at your sample app in Xcode, and I see there a reference to OAuth2.xcodeproj, but as a folder. If I add that file to my project, it stays a file. Even if I drag the folder from the sample app it turns into a file. In either case, when I navigate to Build Phases > Target Dependencies I don't see Oauth2 as it is in the sample app.

I'm relatively new to iOS/Xcode, so hopefully I'm not missing something obvious. I've googled to no avail.

Thanks!

**Comment by p2** [comment]
Hi Hector

I should write a nice documentation, for now I hope the following helps. You should add OAuth2 as a submodule to your project, if you don't use git for your project just clone the library and add it, like so:

- Checkout the repository to download the latest codebase:
        
        $ git clone https://github.com/p2/OAuth2.git

- Open your app project in Xcode and add OAuth2.xcodeproj
- Add OAuth2 as build dependency (look at OAuth2App's build settings to see what and where)

**Comment by hectoroso** [comment]
That's what I did, except that the OAuth2.xcodeproj I added to my project wasn't expandable, with the text underneath reading "3 targets", and Oauth2 wasn't showing up as a build dependency.

However, after some trial and error, I did manage to get it to work by adding the whole OAuth2 folder to my project, then moving the file OAuth2.xcodeproj out of there into the same level as the folder and then removing the folder. It now looks like it does in your OAuth2App sample app and I could add the dependency.

I was then able to build an iOS version of your sample app successfully connecting to GitHub. Thanks!
